components/director/internal/domain/onetimetoken/service.go

```diff
@@ -218,3 +244,44 @@ func (s *service) getTokenFromAdapter(ctx context.Context, adapterURL string, ap
 		Token: externalToken,
 	}, nil
 }
+
+func (s *service) getSuggestedTokenForApp(ctx context.Context, app *model.Application, oneTimeToken *model.OneTimeToken) string {
+	if !tokenSuggestionEnabled(ctx, s.suggestTokenHeaderKey) {
+		return oneTimeToken.Token
+	}
+
+	if app.IntegrationSystemID != nil {
+		log.C(ctx).Infof("Application belongs to an integration system, will use the actual token")
```

Comment:
All of these -> Debug

Reply:
Whatever we choose, the log should include the app guid at least